src/components/NodeEditor/NodeEditor.tsx

```diff
@@ -78,16 +83,95 @@ export interface NodeEditorProps {
     onUpdateLocalizations: Function;
     onUpdateAction: Function;
     onUpdateRouter: Function;
-    /** Perform when editor is closed */
     onClose?(canceled: boolean): void;
     ComponentMap: ComponentMap;
 }
 
 export interface NodeEditorState {
     config: Type;
     show: boolean;
+    setResultName: boolean;
```

Comment:
This PR moves `setResultName` to `NodeEditorState` since multiple forms toggle it. I imagine we'll also be moving the result name value itself to `NodeEditorState` in the future to fix #123